sdk/java/src/main/java/com/chain/http/BatchResponse.java

```diff
@@ -34,7 +34,7 @@ public BatchResponse(Response response, Gson serializer, Type tClass)
         JsonElement elem = root.get(i);
 
         // Test for interleaved errors
-        APIException err = serializer.fromJson(elem, APIException.class);
+        APIException err = serializer.fromJson(elem, eClass);
```

Comment:
My sense is that we could probably reduce the surface area of this change by peeking at the code in the response and check for CH706, but since we'd like to get this out soon this is fine for now.